files/build_templates/docker_image_ctl.j2

```diff
@@ -228,6 +248,7 @@ start() {
         -v /etc/sonic/frr/$DEV:/etc/frr:rw \
 {%- endif %}
         -v /var/run/redis$DEV:/var/run/redis:rw \
```

Comment:
>-v /var/run/redis$DEV:/var/run/redis:rw [](start = 8, length = 39)

Is this line redundant?

Reply:
We would still need this line to get the mappings like /var/run/redis0:/var/run/redis in namespace0 dockers, /var/run/redis1:/var/run/redis in namespace1 dockers etc. 

This is because in places like for eg: python connector classes we refer config files with the generic path like "/var/run/redis/sonic-db/database_config.json".